libvmaf/src/predict.h

```diff
@@ -22,6 +22,12 @@
 #include "feature/feature_collector.h"
 #include "model.h"
 
+int find_linear_function_parameters(Point p1, Point p2,
+                                    double *alpha, double *beta);
+
+int piecewise_linear_mapping(double x, Point *knots, unsigned n_knots,
+                             double *y);
+
```

Comment:
Include the `predict.c` file directly in your test, instead of making these functions public.